packages/browser-runtime-core/src/interpreter/preprocessor/wrap-object-literal.spec.ts

```diff
@@ -18,10 +18,10 @@ describe('wrapObjectLiteral', () => {
     expect(wrapObjectLiteral('\n    {x: 2}  \n ')).to.equal('(\n    {x: 2}  \n )');
   });
 
-  it.skip('should not wrap multiple statements', () => {
-    // TODO: for some reason this is the default behaviour in node repl and devtools,
-    // as is now it wraps everything, which breaks the code.
-    expect(wrapObjectLiteral('{x: 2}; {x: 2}')).to.equal('{x: 2}; {x: 2}');
-  });
+  // it('should not wrap multiple statements', () => {
+  //   // TODO: for some reason this is the default behaviour in node repl and devtools,
+  //   // as is now it wraps everything, which breaks the code.
+  //   expect(wrapObjectLiteral('{x: 2}; {x: 2}')).to.equal('{x: 2}; {x: 2}');
```

Comment:
super corner case, again we can keep it or drop it, maybe give it a look

Reply:
I think this can go under the ticket https://jira.mongodb.org/browse/MONGOSH-256